### ISSUE TITLE
pwndbg "Exception: Cannot add command ctx-watch: already exists."

### ISSUE THREAD

**Issue by SpaceMoehre** [pwndbg/pwndbg#722]
### Describtion

I freshly installed pwndbg but every time I start gdb, I run in the same pwngdb error. I'm pretty new to this, so let me know what else you need. It says that a command already exists but I don't know how to remove it.

### Error
```
Traceback (most recent call last):
  File "/opt/pwndbg/gdbinit.py", line 36, in <module>
    import pwndbg # isort:skip
  File "/opt/pwndbg/pwndbg/__init__.py", line 24, in <module>
    import pwndbg.commands.context
  File "/opt/pwndbg/pwndbg/commands/context.py", line 179, in <module>
    def contextunwatch(expression):
  File "/opt/pwndbg/pwndbg/commands/__init__.py", line 303, in __call__
    _ArgparsedCommand(self.parser, function, alias)
  File "/opt/pwndbg/pwndbg/commands/__init__.py", line 273, in __init__
    super(_ArgparsedCommand, self).__init__(function, command_name=command_name, *a, **kw)
  File "/opt/pwndbg/pwndbg/commands/__init__.py", line 59, in __init__
    raise Exception('Cannot add command %s: already exists.' % command_name)
Exception: Cannot add command ctx-watch: already exists.
```

### System
```
# uname -a
Linux kali 5.4.0-kali4-amd64 #1 SMP Debian 5.4.19-1kali1 (2020-02-17) x86_64 GNU/Linux
```

**Comment by giantbranch** [comment]
Just comment the line 178.
Your number of lines may be different, just search "ctx-watch" in the file.

```
# vim /XXXXX/pwndbg/pwndbg/commands/context.py

170 parser.add_argument("expression", type=str, help="The expression to be evaluated and shown in context")
171 @pwndbg.commands.ArgparsedCommand(parser, aliases=['ctx-watch', 'cwatch'])
172 def contextwatch(expression, cmd=None):
173     expressions.add((expression, expression_commands.get(cmd, gdb.parse_and_eval)))
174
175 parser = argparse.ArgumentParser()
176 parser.description = """Removes an expression previously added to be watched."""
177 parser.add_argument("expression", type=str, help="The expression to be removed from context")
178 #@pwndbg.commands.ArgparsedCommand(parser, aliases=['ctx-watch', 'cunwatch'])
```

**Comment by GrosQuildu** [comment]
@disconnect3d @jerdna-regeiz seems like #711 broke pwndbg ;p

Fixed in #725, please merge.

**Comment by disconnect3d** [comment]
Fixed, sorry for the trouble ;)

Thanks @GrosQuildu
